I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1736, in script
    n "The Golem followed Julian's trail through the sewers, exiting to the city streets. [pronoun] found [pronous] purpose and [pronous] objective."
NameError: name 'pronous' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 1736, in script
    n "The Golem followed Julian's trail through the sewers, exiting to the city streets. [pronoun] found [pronous] purpose and [pronous] objective."
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\ast.py", line 2579, in execute
    Say.execute(self)
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\ast.py", line 621, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\exports\sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\character.py", line 1440, in __call__
    what = self.prefix_suffix("what", self.what_prefix, what, self.what_suffix)
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\character.py", line 1354, in prefix_suffix
    return (sub(prefix) + sub(body) + sub(suffix))
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\character.py", line 1334, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\substitutions.py", line 373, in substitute
    s = interpolate(s, variables) # type: ignore
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\substitutions.py", line 86, in interpolate
    raise e
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\substitutions.py", line 78, in interpolate
    value = renpy.python.py_eval(code, {}, scope)
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Rachel\Desktop\dances\dancingwithshadows\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "<none>", line 1, in <module>
NameError: name 'pronous' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.4.24120703
dancing with shadows draft 1.0
Fri Feb  7 12:06:37 2025
